src/app/screens/sendStx/index.tsx

```diff
@@ -73,7 +73,7 @@ function SendStxScreen() {
   }, [data]);
 
   const handleBackButtonClick = () => {
-    navigate(-1);
+    navigate('/');
```

Comment:
back buttons should all navigate(-1), because we don't assume the sendStx screen has only one entrypoint. recently there was another bug to fix the accountHeader back buttons.

if the bug is the looping back from: https://linear.app/xverseapp/issue/ENG-2678/navigation-issue-when-sending-stx, then probably the confirmStx screen needs to navigate(-1) instead of navigate(/send-stx). 

you can think of the navigation state as a stack, where the back button should be popping off the top instead of putting another on

Reply:
the reason why navigate(/send-stx) was set since we want to pass the state from confirm screen to send screen. If you set navigate(-1) then from my understanding, the previous state cannot be fetched. In the documentation it's written that

Reply:
ok, thanks for looking into that. 

can you put a very short comment in the code to give some context here, explaining why we take this shortcut, even if it's less robust